pycine/cine.py

```diff
@@ -3,7 +3,8 @@
 http://starship.python.net/crew/theller/ctypes/old/codegen.html
 """
 
-from ctypes import *
+from ctypes import c_uint8, c_int16, c_uint16, c_int32, c_uint32, c_char, c_int
+from ctypes import c_int64, c_uint64, c_float, c_double, POINTER, Structure
```

Comment:
You might want to write this like so instead of importing twice from the same module:
```suggestion
from ctypes import (
    c_uint8,
    c_int16,
    c_uint16,
    c_int32,
    c_uint32,
    c_char,
    c_int,
    c_int64,
    c_uint64,
    c_float,
    c_double,
    POINTER,
    Structure,
)
```